main.go

```diff
@@ -55,6 +56,8 @@ var (
 	configMesh                 = flag.String("config-mesh-experimental", "", "Dictates which Mesh resource to use. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
 	includeFederationSupport   = flag.Bool("include-federation-support-experimental", false, "whether or not to generate configs required for xDS Federation. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
 	includeDirectPathAuthority = flag.Bool("include-directpath-authority-experimental", false, "whether or not to include DirectPath TD authority for xDS Federation. Ignored if not used with include-federation-support-experimental flag. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
+	// TODO: default to true when TD supports xdstp style names.
+	includeXDSTPNameInLDS = flag.Bool("include-xdstp-name-in-lds-experimental", false, "whether or not to use XDSTP style name for listener resource name template. Ignored if not used with both include-federation-support-experimental flag. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
```

Comment:
Nit: s/Ignored if not used with both include-federation-support-experimental flag/Ignored if not used with --include-federation-support-experimental flag/

Reply:
Done